jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/utils/SqlNodesCreator.java

```diff
@@ -0,0 +1,76 @@
+package utils;
+
+import com.exasol.adapter.metadata.*;
+import com.exasol.adapter.sql.*;
+
+import java.math.*;
+import java.util.*;
+
+import static com.exasol.adapter.sql.AggregateFunction.AVG;
+
+/**
+ * This class contains static methods for fast creation of SQL nodes which are used in tests for SQL generation
+ * visitors. Helps to avoid duplication and speed up testing.
+ */
+public class SqlNodesCreator {
+    private SqlNodesCreator() {
+    }
+
+    public static SqlOrderBy createSqlOrderByDescNullsFirst() {
+        final List<SqlNode> orderByArguments = new ArrayList<>();
+        final ColumnMetadata columnMetadata = ColumnMetadata.builder().name("test_column").type(DataType.createBool())
+                .build();
+        final ColumnMetadata columnMetadata2 = ColumnMetadata.builder().name("test_column2")
+                .type(DataType.createDouble()).build();
+        final List<Boolean> nulls = new ArrayList<>();
+        nulls.add(false);
+        nulls.add(true);
+        orderByArguments.add(new SqlColumn(1, columnMetadata));
+        orderByArguments.add(new SqlColumn(2, columnMetadata2));
+        return new SqlOrderBy(orderByArguments, nulls, nulls);
```

Comment:
I know this code existed before already, but it is really hard to read.

Reply:
fixed